test/AngularSpec.js

```diff
@@ -435,6 +435,95 @@ describe('angular', function() {
   });
 
 
+  describe('jq', function() {
+    var element;
+
+    beforeEach(function() {
+      element = document.createElement('html');
+    });
+
+    afterEach(function() {
+      jq.name_ = undefined;
+      delete jq.name_;
```

Comment:
@gkalpak Sorry, force of habit from a time when delete was buggy in javascript.  I guess that's fixed now since Angular supports IE 9 and above.